src/components/crossDevice/MobileConnected/assets/connection-large.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I think Stefania knows how to optimize these

Reply:
I use this tool https://jakearchibald.github.io/svgomg/

Reply:
optimise for size? If so we should do that for all the icons (in a separate issue)

Reply:
Yes, I did it for all the icons in my previous PR because most of them had repeated lines of code but yes, we should definitely check which svgs need to be optimised.